lib/crowbar/client/app/upgrade.rb

```diff
@@ -219,12 +219,12 @@ def backup(component)
           catch_errors(e)
         end
 
-        desc "repocheck ADDON",
-          "Check for existing repositories for an addon (ha|storage)"
+        desc "repocheck COMPONENT",
+          "Check for existing repositories for a component (crowbar|ha|storage)"
```

Comment:
> We do not have anything like crowbar repository.

in fact we're upgrading the admin (crowbar) node, that's why i think it makes sense to call it `crowbar`.